keyboards/gray_studio/think65v3/readme.md

```diff
@@ -0,0 +1,2 @@
+# Gray Studio think65v3
+
```

Comment:
This file needs to follow the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md).